src/trade/tradeDialog.scss

```diff
@@ -108,7 +108,7 @@
     }
     .trade-template-manager-root {
         position: absolute;
-        margin: -2px 0 0 4px;
+        margin: -2px 0 0 -2px;
```

Comment:
I'm still able to open the underlying dropdown through the disabled `save changes`, could you take a look again?